src/main/groovy/de/triplet/gradle/play/PlayPublishTask.groovy

```diff
@@ -47,4 +49,14 @@ class PlayPublishTask extends DefaultTask {
         }
     }
 
+    def getTrackByName(String trackName) {
```

Comment:
Before fixing this code, I would like you guys ( @ChristianBecker @bhurling ) discuss about where this plugin goes.

As far as I remember, @bhurling didn't want to add some features that are not related with gradle build process (#108 and #110). Here is the comments.

https://github.com/Triple-T/gradle-play-publisher/issues/109#issuecomment-152189289
https://github.com/Triple-T/gradle-play-publisher/issues/109#issuecomment-157172738

If you guys decide merging this in, I would fix this and the conflicts as soon as possible.